src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/EnumGroupAndName.cs

```diff
@@ -10,8 +14,14 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
     /// </summary>
     public struct EnumGroupAndName
     {
+        private DisplayAttribute _displayAttribute;
+        private IStringLocalizer _stringLocalizer;
+        private FieldInfo _fieldInfo;
+        private string _name;
+
         /// <summary>
-        /// Initializes a new instance of the EnumGroupAndName structure.
+        /// Initializes a new instance of the EnumGroupAndName structure. This constructor should not be used in any 
+        /// site where users may select a locale.
```

Comment:
My concern with "multiple locales may be in use" is that it sounds like you're allowing a single user to have multiple simultaneous locals rather than the option between a number of them.